alacritty_terminal/src/config/scrolling.rs

```diff
@@ -26,8 +27,9 @@ impl Scrolling {
         self.multiplier.0
     }
 
-    pub fn faux_multiplier(self) -> u8 {
-        self.faux_multiplier.0
+    // TODO: DEPRECATED
```

Comment:
Adding this comment just on the field should be enough. All other instances are automatically brought to our attention by Rust's typesystem once the field is removed :fireworks:.